c2corg_ui/templates/outing/edit.html

```diff
@@ -90,7 +90,9 @@ <h2 class="heading show-phone" translate>general informations</h2>
 
             ## ACTIVITIES
             <div class="form-group full" id="outing-activities" ng-init="activities = ${activities}">
-              <label><span translate>activities</span> *</label>
+              <label app-context-help ="context_help_activities" context-help-title="activities">
```

Comment:
* Extra blank space before the = sign.
* Can't you use a translate filter instead of adding the string to ``c2corg_ui/templates/i18n.html``? ``app-context-help="{{'context_help_activities' | translate}}"``